packages/channels/botpress-channel-messenger/src/index.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
     enabled: { type: 'bool', required: true, default: true },
     hostname: { type: 'string', required: false, default: '', env: 'MESSENGER_HOST' },
 
+    graphVersion: { type: 'string', required: true, default: '2.12' },
```

Comment:
It's 2.12 here but 2.11 in config.json above. Can we keep it consistent?
Does it work fine with this version?

Reply:
@mrbarletta , no let's keep it at 2.12. But I'd say we should keep same version in config.json either.

Reply:
done! -  set 2.12 in both files